keyboards/yellowbrickroad/keymaps/led/rules.mk

```diff
@@ -0,0 +1,7 @@
+RGB_MATRIX_ENABLE = yes       # Use RGB matrix (Don't enable this when RGBLIGHT_ENABLE is used.)
+RGB_MATRIX_DRIVER = ws2812    # 2021/01/23 added.
+RGB_MATRIX_CUSTOM_KB = yes    #
```

Comment:
These should be moved to the keyboard level.  In fact, the driver is, already. 

```suggestion
```

Reply:
Done.